src/main/kotlin/service/TradingService.kt

```diff
@@ -0,0 +1,20 @@
+package service
+
+import domain.CryptoExchange
+import domain.User
+import domain.Wallet
+import java.math.BigDecimal
+import domain.Currency
+
+interface TradingService {
```

Comment:
I understand that we all are were taught to create interfaces even when dealing with Spring, but in this case these interfaces for services are redundant. Try not to overload packages with interfaces which will be implemented only once